app/helpers/service_request_helper.rb

```diff
@@ -16,4 +16,10 @@ def cents_to_money(cents)
     Money.new(cents)
   end
   
+  def estimation_dies(dies_at)
+    dies_at = Time.parse dies_at
+    Time.now > dies_at ? true : false
```

Comment:
It's better to use Time.current in rails projects so it will use configured Timezone in app. Time.now will use server timezone which can lead to errors.